specs/v1.yaml

```diff
@@ -61,11 +61,13 @@ paths:
           $ref: "#/components/responses/InternalServerError"
   /v1/nonprofits:
     post:
+      deprecated: true
       summary: Create nonprofit
       description: |-
         Create a nonprofit organization account.
         This is useful for integration partners to use after a nonprofit consents to use the Chariot payment option on their donation forms.
         If a nonprofit does not already exist for the EIN, this will return a 201 Created status.
+        If a nonprofit does not exist in GuideStar.org, this will return a 404 Not Found status.
         If a nonprofit already exists for the given EIN on the system, this will return a 409 Conflict status.
```

Comment:
```suggestion
        If a nonprofit already exists for the given EIN on the system, this will return a 409 Conflict status.
        If a nonprofit exists in GuideStar.org but they are not pub78 verified, this will return a 412 Precondition Failed status
```